README.md

```diff
@@ -156,7 +156,22 @@ cd frontend && npm run dev
 Currently, the environment variable `VITE_APP_USE_STREAMING` is specified on the frontend side. It's recommended to set it to `false` when running the backend locally and `true` when operating on AWS.  
 When streaming is enabled, text is generated in real-time due to the streaming of content generation results.
 
+### Local development using docker compose
 
+[docker-compose.yml](./docker-compose.yml) allows you to run and develop frontend/backend APIs/DynamoDB Local in your local environment.
+
+※ The hot reloading is only supported on the frontend, not on the backend API. It's depends on the directory structure which prevents mount the source code.
```

Comment:
細かいところで恐れ入りますが、

※ Hot reloading is only supported on the frontend, not on the backend API because the source code cannot be mounted due to the directory structure.

などの表現の方が自然かなと思いました。

Reply:
ありがとうございます！
修正しました！